src/localization/tableStrings.js

```diff
@@ -50,45 +50,51 @@ export const tableStrings = new LocalizedStrings({
     supply_quantity: 'SUPPLY THIS\nINVOICE',
     their_stock: 'THEIR\nSTOCK',
   },
-  gil: { // TODO add: batch, batch_expiry, our_stock, supply_quantity, supplier
-    actual_quantity: 'Iraua\nMwaitina',
-    available_stock: 'Mwaitin ae\niai ngkai',
+  gil: {
+    actual_quantity: 'IRAUA\nMWAITINA',
+    available_stock: 'MWAITIN AE\nIAI NGKAI',
+    batch_expiry: 'BANEN TAINA',
     category: 'Kurubu',
     code: 'CODE',
-    comment: 'Iai riki?',
-    created_date: 'Bong -  karaoaia iai',
-    current_stock: 'Mwaitin ae\niai ngkai',
-    customer: 'Kiriniki/Aran te aoraki',
+    comment: 'IAI RIKI?',
+    created_date: 'BONG -\nKARAOAIA IAI',
+    current_stock: 'MWAITIN AE\nIAI NGKAI',
+    customer: 'KIRINIKI/ARAN TE AORAKI',
     delete: 'DELETE',
     department: 'Tabo',
-    difference: 'Kaokorona',
+    difference: 'KAOKORONA',
     earliest_expiry: 'Earliest expiry',
-    entered_date: 'Bong - karinaki iai',
+    entered_date: 'BONG -\nKARINAKI IAI',
     finalised: 'Finalised ',
     id: 'ID',
     in_progress: 'Ae karaoaki',
     invoice_number: 'INVOICE\nNUMBER',
     invoices: 'INVOICES',
-    item_code: 'Ana code',
-    item_name: 'Arana ',
+    item_code: 'ANA CODE',
+    item_name: 'ARANA ',
     items: 'ITEMS',
     monthly_usage: 'MONTHLY USE',
-    name: 'Arana',
+    name: 'ARANA',
     no_change: 'Akea te bitaki',
     number_of_batches: 'Mwatin taian batch',
-    number_received: 'Mwaitin ae reke raoi',
-    number_sent: 'Mwaitin ae kanakoaki',
-    quantity: 'Mwaitina',
+    number_received: 'MWAITIN AE REKE RAOI',
+    number_sent: 'MWAITIN AE KANAKOAKI',
+    our_stock: 'MAITIN ARA BWAIN AORAKI INANON BAIRA',
```

Comment:
Does our_stock fit in the table ?

Reply:
Good questions. I think I missed those ones. Let me check it now...

Reply:
I think it's good. What do you think?
Here it is, English and Gilbertese versions:
![image](https://user-images.githubusercontent.com/16461988/31525972-7bfdf144-b020-11e7-8f67-bbc7eb6c4cda.png)